lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Sorry, I mean which setting or provider's triggering it?  It seems like we have a few settings in providers that could be nil by default.